docs/advanced.md

```diff
@@ -155,21 +155,23 @@ You can extend the K3s base template instead of copy-pasting the complete stock
   BinaryName = "/usr/bin/custom-container-runtime"
 
 ```
-## NVIDIA Container Runtime Support
 
-K3s will automatically detect and configure the NVIDIA container runtime if it is present when K3s starts.
+## NVIDIA Container Toolkit Support
+
+K3s will automatically detect and configure the NVIDIA container toolkit if it is present when K3s starts.
 
 1. Install the nvidia-container package repository on the node by following the instructions at:
-    https://nvidia.github.io/libnvidia-container/
-1. Install the nvidia container runtime packages. For example:
-   `apt install -y nvidia-container-runtime cuda-drivers-fabricmanager-515 nvidia-headless-515-server`
-1. Install K3s, or restart it if already installed:
+    <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html>
+2. Install the nvidia container runtime packages. For example:
+   `apt-get install -y nvidia-container-toolkit cuda-drivers-fabricmanager-515 nvidia-headless-515-server`
+3. Install K3s, or restart it if already installed:
     `curl -ksL get.k3s.io | sh -`
-1. Confirm that the nvidia container runtime has been found by k3s:
+4. Confirm that the nvidia container runtime has been found by k3s:
```

Comment:
there appear to be a bunch of unrelated changes to whitespace and the list numbering. These don't affect the actual rendered doc content. Can you trim your changes down to just the link change and package name?